src/api.js

```diff
@@ -0,0 +1,11 @@
+export default class Api {
+  static async moveToApi(player) {
+    await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Ml0wfxegyFf06umHiVlI/scores/', {
+      method: 'POST',
+      headers: {
+        'Content-type': 'application/json',
+      },
+      body: JSON.stringify(player),
+    }).json();
```

Comment:
- [ ] Kindly ensure that a user can post data. This method throws an error `json is not a function` in the browser console. Kindly fix it.